src/components/AddPerson/AddPerson.tsx

```diff
@@ -0,0 +1,218 @@
+import React, { useState } from 'react';
+import { Person } from '../../types/Person';
+
+type Props = {
+  people: Person[],
+  addPerson: (value: Person) => void,
+};
+
+export const AddPerson: React.FC<Props> = ({ people, addPerson }) => {
+  const [personName, setPersonName] = useState('');
+  const [personLastName, setPersonLastName] = useState('');
+  const [personSex, setPersonSex] = useState('m');
+
+  const [personBorn, setPersonBorn] = useState('');
+  const [personDied, setPersonDied] = useState('');
+  const [personFather, setPersonFather] = useState('');
+  const [personMother, setPersonMother] = useState('');
```

Comment:
Let's use one state with an object that will contain all needed properties. It's hard to manage 7 states🤯